src/ert/storage/local_storage.py

```diff
@@ -300,29 +299,26 @@ def _save_index(self) -> None:
     def _migrate(self, ignore_migration_check: bool) -> None:
         if ignore_migration_check:
             return
+        from ert.storage.migration import (  # noqa: PLC0415
```

Comment:
Are we sure this will work in all circumstances? Considering the exisiting "`disable=C0415`" that is now removed.

Reply:
Yes, that was the pylint equivalent, which we no longer use.